frontend/src/components/search/SearchFilterAccordion/SearchFilterAccordion.tsx

```diff
@@ -116,7 +116,7 @@ export function SearchFilterAccordion({
       content: getAccordionContent(),
       expanded: false,
       id: "funding-instrument-filter",
-      headingLevel: "h4",
+      headingLevel: "h2",
```

Comment:
This did not increase the size surprisingly - there's some other css that overrides this I think - I didn't investigate further since it just works here

Reply:
Oh! Yeah, this component lets you use whatever heading level you want, but styles them all the same (as consistently-sized accordion headings)